packages/manager/apps/hycu/src/pages/listing/Listing.page.tsx

```diff
@@ -38,6 +39,16 @@ import HycuActionMenu from './menu/HycuActionMenu.component';
 import { getStatusColor } from '@/utils/statusColor';
 
 /* ========= datagrid cells ========= */
+const DatagridErrorCell = () => {
+  const { t } = useTranslation('hycu/listing');
+
+  return (
+    <DataGridTextCell>
+      <OsdsText>{t('hycu_listing_error_loading_data')}</OsdsText>
+    </DataGridTextCell>
```

Comment:
should be displayed in error policy, right ?

Reply:
No we want in grey because it's not impactant